nodes/src/nodes/tool_git/IInstance.py

```
@@ -0,0 +1,432 @@
+        'name': 'git.status',
+        'description': 'Return the working-tree status: current branch, staged files, unstaged modifications, and untracked files.',
+        'input_schema': {'type': 'object', 'properties': {}, 'required': []},
+    },
+    {
+        'name': 'git.log',
+        'description': 'Return commit history. Supports filtering by branch, file path, author name, and date range.',
+        'input_schema': {
+            'type': 'object',
+            'required': [],
+            'properties': {
+                'max_count': {'type': 'integer', 'description': 'Maximum number of commits to return (1–200, default 20).'},
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)